src/platform/plugins/shared/dashboard/server/content_management/v1/cm_services.ts

```diff
@@ -264,10 +342,12 @@ export const dashboardAttributesSchema = searchResultsAttributesSchema.extends({
 
   // Dashboard Content
   controlGroupInput: schema.maybe(controlsGroupSchema),
-  panels: schema.arrayOf(schema.oneOf([panelSchema, sectionSchema]), { defaultValue: [] }),
+  panels: schema.maybe(
```

Comment:
Does panels need to be optional? Any optional schemas (those wrapped with `schema.maybe`) should not have a `defaultValue` as the default values will not be applied to optional schemas. 

🎩  to @nickofthyme who recently informed me of this